language/move-stdlib/sources/FixedPoint32.move

```diff
@@ -188,20 +188,20 @@ module Std::FixedPoint32 {
         aborts_if quotient == 0 && scaled_numerator != 0 with Errors::INVALID_ARGUMENT;
         aborts_if quotient > MAX_U64 with Errors::LIMIT_EXCEEDED;
     }
-    spec fun spec_concrete_create_from_rational(numerator: num, denominator: num): FixedPoint32 {
+    spec fun spec_create_from_rational(numerator: num, denominator: num): FixedPoint32 {
         FixedPoint32{value: (numerator << 64) / (denominator << 32)}
     }
 
     /// # Abstract Semantics
 
-    spec schema CreateFromRationalAbortsIf {
+    spec schema AbstractCreateFromRationalAbortsIf {
         /// This is currently identical to the concrete semantics.
-        include ConcreteCreateFromRationalAbortsIf;
+        include CreateFromRationalAbortsIf;
```

Comment:
Also we could remove these, if we are going down the direction of removing the abstractions.

Reply:
Done.